themes/socialbase/templates/group/group--default.html.twig

```diff
@@ -41,8 +41,22 @@
 #}
 {% if content.field_group_description|render %}
   <div{{ attributes.addClass('card') }}>
+
+    {%
+      set title_classes = [
+      'field--label',
+      label_display == 'visually_hidden' ? 'sr-only',
+    ]
+    %}
+
     <div{{ content_attributes.addClass('card__body body-text') }}>
-      {{ content.field_group_description }}
+      {% if label_hidden %}
+        {{ content.field_group_description }}
+      {% else %}
+        <div{{ title_attributes.addClass(title_classes) }}>{{ label }}</div>
+
+        {{ content.field_group_description }}
+      {% endif %}
```

Comment:
We should just render the label if it's not hidden and not duplicate the `{{ content.field_group_description }}`

Reply:
Fixed